pkgs/development/libraries/webkitgtk/default.nix

```diff
@@ -143,6 +173,10 @@ stdenv.mkDerivation rec {
   ]) ++ lib.optionals stdenv.isDarwin [
     libedit
     readline
+    # pull a header that contains a definition of proc_pid_rusage
+    (runCommandNoCC "${pname}_headers" {} ''
```

Comment:
Why not add the whole `sdk`?

Reply:
Because that won't work. It seems like the sdk as a whole is not compatible with our libc++.

Reply:
How about adding a comment about that. Just to keep the knowledge there, so the next person knows why it's been done so.

Reply:
Added the comment